services/core-web/src/components/Forms/EMLIContacts/EMLIContactForm.js

```diff
@@ -0,0 +1,321 @@
+        )}
+
+        <Row gutter={16}>
+          <Col span={24}>
+            <Form.Item>
+              <Field
+                id="email"
+                name="email"
+                label="Email*"
+                component={renderConfig.FIELD}
+                validate={[email, required]}
+                disabled={!props.formValues.emli_contact_type_code}
+              />
+            </Form.Item>
+          </Col>
+        </Row>
+        <Row gutter={16}>
+          <Col span={24}>
+            <Form.Item>
+              <Field
+                id="phone_number"
+                name="phone_number"
+                label="Phone Number*"
+                placeholder="e.g. xxx-xxx-xxxx"
+                component={renderConfig.FIELD}
+                validate={[required, phoneNumber, maxLength(12)]}
+                normalize={normalizePhone}
+                disabled={!props.formValues.emli_contact_type_code}
+              />
+            </Form.Item>
+          </Col>
+        </Row>
+        {officeCodes.includes(props.formValues.emli_contact_type_code) && (
+          <>
+            <Row gutter={16}>
+              <Col span={24}>
+                <Form.Item>
+                  <Field
+                    id="fax_number"
+                    name="fax_number"
+                    label="Fax Number"
+                    placeholder="e.g. xxx-xxx-xxxx"
+                    component={renderConfig.FIELD}
+                    validate={[phoneNumber, maxLength(12)]}
+                    normalize={normalizePhone}
+                    disabled={!props.formValues.emli_contact_type_code}
+                  />
+                </Form.Item>
+              </Col>
+            </Row>
+            <Row gutter={16}>
+              <Col span={24}>
+                <Form.Item>
+                  <Field
+                    id="mailing_address_line_1"
+                    name="mailing_address_line_1"
+                    label={
+                      props.formValues.emli_contact_type_code === regionalOfficeCode
+                        ? "Mailing Address Line 1*"
+                        : "Mailing Address Line 1"
+                    }
+                    component={renderConfig.AUTO_SIZE_FIELD}
+                    validate={
+                      props.formValues.emli_contact_type_code === regionalOfficeCode
+                        ? [required]
+                        : []
+                    }
+                    disabled={!props.formValues.emli_contact_type_code}
+                  />
+                </Form.Item>
+              </Col>
+            </Row>
+            <Row gutter={16}>
+              <Col span={24}>
+                <Form.Item>
+                  <Field
+                    id="mailing_address_line_2"
+                    name="mailing_address_line_2"
+                    label={
+                      props.formValues.emli_contact_type_code === regionalOfficeCode
+                        ? "Mailing Address Line 2*"
```

Comment:
is there anywhere on this form where the asterisk used here is explained?

Reply:
No, The asterisk just means required. 
So the address is required for Regional offices, not for the MM office. Additionally, the address is only visible to the two office type codes. 

@JustinMacaulay does that answer your question?